hawkular-alerts-commons/src/main/java/org/hawkular/alerts/cache/IspnCacheManager.java

```diff
@@ -34,6 +37,8 @@
  */
 public class IspnCacheManager {
     private static final MsgLogger log = MsgLogging.getMsgLogger(IspnCacheManager.class);
+    private static final String ISPN_BACKEND_REINDEX = "hawkular-alerts.backend-reindex";
+    private static final String ISPN_BACKEND_REINDEX_DEFAULT = "false";
```

Comment:
Perhaps we can set default true, but that's minor, having the ability to configure this is good.